### ISSUE TITLE
GSoC final list of tasks

### ISSUE THREAD

**Issue by eimrek** [aiidateam/aiida-explorer#7]
The project is progressing well, but to have a nice final state as the Google Summer of Code project, let's collect here a list of tasks that would be great to have implemented:

### General

- [x] fix routing issues (when refreshing, when directly linking to a node, e.g. https://aiida-explorer.netlify.app/mc3d/details/29af2b6a-7929-4f75-9b53-399721b8ae8a)
- [x] #6
- [x] statistics tab
- [ ] #8 

### Node grid

- [x] Disable filter textboxes;
- [x] Implement sorting similar to the old implementation;
- [x] fix computers;
- [x] loading animation;
- [x] better pagination controls;

### Node details

- [x] move node metadata outside raw view (see #6)
- [x] Don't show `"root"` in raw view
- [x] Node extras: only show data.extras
- [x] Calcjobnodes: fix download and preview of output files (see the old implementation);
- [ ] #9
- [x] Rich view: CalcFunctionNode
  - [old example](https://www.materialscloud.org/explore/mc3d/details/262b1ebd-ca63-48e5-b78f-6b20a4a9e1f8)
  - basically just display the python code.

### Graph browser

- [x] Fix coloring and shape:
  - all nodes rectangular for now.
  - Data - light green, Calcjobs - light red, Workflows - orange.
- [x] fix hover overlay blocking the node;
- [ ] #12
- [ ] #11

Note: things like the "History view" or "Creator hierarchy" should not be prioritized.

@edan-bainglass @superstar54 feel free to add extra stuff that i missed.

**Comment by eimrek** [comment]
As the GSOC project is finished now, I am closing this issue and converted any remaining tasks to their separate issues. Thanks @SharanRP!
